### PR TITLE
Added feature 1

### PR DESCRIPTION
**Pull request by rvsubagan**
Hey, I just added a feature 1 module. Please check it out!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,5 +4,9 @@
     </head>
     <body>
         <h1>Nami App 2</h1>
+
+        <module feature1>
+            <h2>Feature 1</h2>
```
Comment (fishmankaratedo) [inline index.html]: Hey @rvsubagan, can you add in closing parenthesis (fork) after the h2 feature1? Thanks!

Comment (fishmankaratedo) [reply index.html]: Okay. Thanks!

Comment (rvsubagan) [reply index.html]: Hey, I've added in the new changes. Please check. Thanks!

**Review by fishmankaratedo — APPROVED** [review]
LGTM!

**Comment by rvsubagan** [comment]
Awesome, thanks!

**Comment by rvsubagan** [comment]
Okay sure. Thanks!

**Pull request merged**